src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+[    2.512386] pci 0000:09:00.0: Adding to iommu group 16 
+[    2.512410] pci 0000:09:00.2: Adding to iommu group 17 
+[    2.512433] pci 0000:09:00.3: Adding to iommu group 18 
+[    2.512455] pci 0000:0a:00.0: Adding to iommu group 19 
+[    2.512479] pci 0000:0a:00.2: Adding to iommu group 20 
+[    2.512503] pci 0000:0a:00.3: Adding to iommu group 21 
+[    2.512712] pci 0000:00:00.2: AMD-Vi: Found IOMMU cap 0x40 
+[    2.513088] perf/amd_iommu: Detected AMD IOMMU #0 (2 banks, 4 counters/bank). 
+[    2.730288] AMD-Vi: AMD IOMMUv2 driver by Joerg Roedel <jroedel@suse.de> 
+```
+
+On an Intel system, you may get an output like the following (courtesy of the
+archwiki):
+
+```
+[    0.000000] ACPI: DMAR 0x00000000BDCB1CB0 0000B8 (v01 INTEL  BDW      00000001 INTL 00000001)
+[    0.000000] Intel-IOMMU: enabled
+[    0.028879] DMAR: IOMMU 0: reg_base_addr fed90000 ver 1:0 cap c0000020660462 ecap f0101a
+[    0.028883] dmar: IOMMU 1: reg_base_addr fed91000 ver 1:0 cap d2008c20660462 ecap f010da
+[    0.028950] IOAPIC id 8 under DRHD base  0xfed91000 IOMMU 1
+[    0.536212] DMAR: No ATSR found
+[    0.536229] IOMMU 0 0xfed90000: using Queued invalidation
+[    0.536230] IOMMU 1 0xfed91000: using Queued invalidation
+[    0.536231] IOMMU: Setting RMRR:
+[    0.536241] IOMMU: Setting identity map for device 0000:00:02.0 [0xbf000000 - 0xcf1fffff]
+[    0.537490] IOMMU: Setting identity map for device 0000:00:14.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537512] IOMMU: Setting identity map for device 0000:00:1a.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537530] IOMMU: Setting identity map for device 0000:00:1d.0 [0xbdea8000 - 0xbdeb6fff]
+[    0.537543] IOMMU: Prepare 0-16MiB unity mapping for LPC
+[    0.537549] IOMMU: Setting identity map for device 0000:00:1f.0 [0x0 - 0xffffff]
+[    2.182790] [drm] DMAR active, disabling use of stolen memory
+```
+
+## Isolating the GPU
+
+Next, we need to isolate the GPU to use the `vfio-pci` rather than the original
+Nvidia or AMD graphics driver.
+
+> Note: Whatever GPU you isolate here will no longer be usable on your host
+> system
+
+### Checking IOMMU groups
+
+Different motherboards and motherboard manufacturers have different ways to
+setup IOMMU grouping. If your IOMMU groups are not proper, you may have to patch
+your kernel (see: [ACS Override
+Patch](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Bypassing_the_IOMMU_groups_(ACS_override_patch))),
+change the PCI slot your GPU is in, or use a different motherboard. Once again,
+the example rigs are helpful here. If someone has accomplished it with your
+motherboard, it is possible.
+
+To check your groups, run the following script:
+
+```
+#!/bin/bash
+shopt -s nullglob
+for g in /sys/kernel/iommu_groups/*; do
+    echo "IOMMU Group ${g##*/}:"
+    for d in $g/devices/*; do
+        echo -e "\t$(lspci -nns ${d##*/})"
+    done;
+done;
+```
+
+Here is the output for the above script on my machine:
```

Comment:
Again, i'd prefer not to see long sections of sample output, but a description of what the user needs to look for to check that things are in order.